publication-commons/src/test/java/no/unit/nva/publication/external/services/PersonApiClientTest.java

```diff
@@ -51,7 +54,7 @@ void shouldReturnPersonsOrgUnitIdsWhenInputIsUserExistingInPersonService()
         throws IOException, InterruptedException, ApiGatewayException {
         personApiClient = new PersonApiClient(createMockHttpClientReturningResponse(successfulNonEmptyResponse()));
         var userAffiliations = personApiClient.fetchAffiliationsForUser(inputFeideId);
-        assertThat(userAffiliations, is(hasItems(URI.create("https://api.cristin.no/v2/units/194.63.10.0"))));
+        assertThat(userAffiliations, is(hasItems(randomOrgUnitId)));
```

Comment:
I don't like then name random here, while the value is randomly generated the value is not random within the test.